packages/node/src/integrations/localvariables/inspector.d.ts

```diff
@@ -3358,6 +3358,30 @@ declare module 'node:inspector' {
   export = inspector;
 }
 
+/**
+ * @types/node doesn't have a `node:inspector/promises` module, maybe because it's still experimental?
```

Comment:
`@types/node` has it, but it is under `inspector`. Ref: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/inspector.d.ts#L1775

Reply:
Thats the sync interface with callbacks:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a84a09dd3a6ce50a136a7c358c8581f46c812cf1/types/node/inspector.d.ts#L1818

The `node:inspector/promises` API returns promises:
https://nodejs.org/api/inspector.html#sessionpostmethod-params